tests/check_public_interfaces.py

```diff
@@ -194,6 +197,18 @@ def test_purepy_ed25519(self):
         pub, 'ed25519', bsig, data)
     self.assertEqual(False, invalid)
 
+  def test_gpg_cmds(self):
+    """Ensure functions calling GPG commands throw an appropriate error"""
+
+    with self.assertRaises(securesystemslib.exceptions.UnsupportedLibraryError):
+      securesystemslib.gpg.functions.create_signature('bar')
+
+    with self.assertRaises(securesystemslib.exceptions.UnsupportedLibraryError):
+      securesystemslib.gpg.functions.export_pubkey('f00')
+
+    with self.assertRaises(securesystemslib.exceptions.UnsupportedLibraryError):
+      securesystemslib.gpg.util.get_version()
+
```

Comment:
Could you add a `verify_signature` call too? IIUC it should also raise an `UnsupportedLibraryError`.

Reply:
Yes I can. That will throw `UnsupportedLibraryError` eventually, from whichever handler's `verify_signature` function. Does it make sense to add an explicit check and error early? That probably only makes the tests cleaner/easier and doesn't affect usage, so perhaps now.

Reply:
> Does it make sense to add an explicit check and error early? 

I wondered too. Probably doesn't hurt. :)

> That probably only makes the tests cleaner/easier and doesn't affect usage, so perhaps now.

Not sure I understand why it makes the tests cleaner.

Reply:
> > Does it make sense to add an explicit check and error early?
> 
> I wondered too. Probably doesn't hurt. :)

👍 

> > That probably only makes the tests cleaner/easier and doesn't affect usage, so perhaps now.
> 
> Not sure I understand why it makes the tests cleaner.

Perhaps not cleaner. But easier/shorter to write, as the arguments can be any type rather than needing to be correct enough to pass the initial parts of the function.